public/docs/_examples/security/ts/app/bypass-security.component.ts

```diff
@@ -8,26 +8,31 @@ import { DomSanitizationService, SafeResourceUrl, SafeUrl } from '@angular/platf
   templateUrl: 'app/bypass-security.component.html',
 })
 export class BypassSecurityComponent {
-  dangerousUrl: SafeUrl;
+  dangerousUrl: string;
+  trustedUrl: SafeUrl;
+  dangerousVideoUrl: string;
   videoUrl: SafeResourceUrl;
 
   // #docregion trust-url
   constructor(private sanitizer: DomSanitizationService) {
-    // javascript: URLs are dangerous if attacker controlled. Angular sanitizes them in data
-    // binding, but we can explicitly tell Angular to trust this value:
-    this.dangerousUrl = sanitizer.bypassSecurityTrustUrl('javascript:alert("Hi there")');
+    // javascript: URLs are dangerous if attacker controlled.
+    // Angular sanitizes them in data binding, but we can
+    // explicitly tell Angular to trust this value:
+    this.dangerousUrl = 'javascript:alert("Hi there")';
```

Comment:
Maybe call out in the text describing this example that Angular will log to the console (in development mode) that it sanitized an unsafe value here?